apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -254,6 +258,22 @@ function wpcom_global_styles_in_use_by_wp_global_styles_post( array $wp_global_s
 	return $global_styles_in_use;
 }
 
+/**
+ * Checks if the current user can edit the `wp_global_styles` post type.
+ *
+ * @param int $blog_id Blog ID.
+ * @return bool Whether the current user can edit the `wp_global_styles` post type.
+ */
+function wpcom_global_styles_current_user_can_edit_wp_global_styles( $blog_id = 0 ) {
+	if ( ! $blog_id ) {
+		$blog_id = get_current_blog_id();
+	}
+	switch_to_blog( $blog_id );
```

Comment:
By the way, this seems to be running on Atomic sites, which means it causes a crash there :(

Reply:
Closing the loop: this has been fixed by https://github.com/Automattic/wp-calypso/pull/73621.
Again apologies for the mess; it was entirely our fault.